homeassistant/components/smlight/strings.json

```
@@ -0,0 +1,68 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set up SMLIGHT Zigbee Integration",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of the SMLIGHT SLZB-06x device"
+        }
+      },
+      "auth": {
+        "description": "Please enter the username and password",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "Please enter the correct username and password for host: {host}",
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      },
+      "confirm_discovery": {
+        "description": "Do you want to set up SMLIGHT at {host}?"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "reauth_failed": "[%key:common::config_flow::error::invalid_auth%]",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "zigbee_temperature": {
+        "name": "Zigbee SoC Temp"
+      },
+      "core_temperature": {
+        "name": "Core SoC Temp"
+      },
+      "fs_usage": {
+        "name": "Filesystem Usage"
+      },
+      "ram_usage": {
+        "name": "RAM Usage"
```

```suggestion
        "name": "RAM usage"
```
use sentence case (also for rest)